crates/viewer/re_view_spatial/src/visualizers/arrows2d.rs

```diff
@@ -264,7 +264,9 @@ impl VisualizerSystem for Arrows2DVisualizer {
                                 .map_or(&[], |class_ids| bytemuck::cast_slice(class_ids)),
                             keypoint_ids: keypoint_ids
                                 .map_or(&[], |keypoint_ids| bytemuck::cast_slice(keypoint_ids)),
-                            show_labels: show_labels.unwrap_or_default().get(0).map(Into::into),
+                            show_labels: show_labels
+                                .map(|b| !b.is_empty() && b.value(0))
+                                .map(Into::into),
```

Comment:
I find the way this was expressed before much more readable... is there no fallible `get()` method in arrow1?

Reply:
Sadly not. All getters in arrow panics rather than return an `Option`.